extensionsGallery-insider.json

```diff
@@ -4651,7 +4651,7 @@
 					"versions": [
 						{
 							"version": "0.2.7",
-							"lastUpdated": "02/28/2023",
+							"lastUpdated": "03/17/2023",
```

Comment:
We already have a 0.2.7 version. Is this supposed to replace the one that's uploaded? If so we can't do that - you have to bump the version and release a new one (otherwise people with the current 0.2.7 version will not get updated unless then uninstall & reinstall)

Reply:
I have bumped up the version to 0.2.8. This is the vsix file: 

https://github.com/microsoft/azuredatastudio-mysql/releases/download/v0.2.8/azuredatastudio-mysql-0.2.8.vsix